scrapper.py

```diff
@@ -27,71 +39,160 @@ class UnknownConfigError(Exception):
+        self.max_articles_per_seed = max_articles_per_seed
+        self.urls = []
 
     @staticmethod
     def _extract_url(article_bs):
-        pass
+        soup_strings = article_bs.find_all(class_="block-content")
+        links = re.findall(r'(\"https://astravolga.ru/(\w+[-])+\w+/")', str(soup_strings))
+        return links
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) '
+                          'Chrome/88.0.4324.182 Safari/537.36 Edg/88.0.705.74'}
+        for url in self.seed_urls:
+            try:
+                response = requests.get(url, headers=headers)
+                if response.status_code == 200:
+                    sleep(random.randrange(3, 6))
+                    soup_menu_page = BeautifulSoup(response.content, 'lxml')
+                else:
+                    raise BadStatusCode
+            except BadStatusCode:
+                continue
+            else:
+                links = self._extract_url(soup_menu_page)
+                articles_per_one_seed = 0
+                for link in links:
+                    if articles_per_one_seed >= self.max_articles_per_seed or len(self.urls) >= self.total_max_articles:
+                        continue
+                    else:
+                        articles_per_one_seed += 1
+                        self.urls.append(link[0][1:-2])
+        return self.urls
 
     def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
 
 
 class ArticleParser:
     """
     ArticleParser implementation
     """
-    def __init__(self, full_url: str, article_id: int):
-        pass
+    def __init__(self, article_url: str, article_id: int):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = article.Article(article_url, article_id)
 
     def _fill_article_with_text(self, article_soup):
-        pass
+        text_soup = article_soup.find('div', class_="entry-content").find_all('p')
+        text = ''
+        for par in text_soup:
+            if par.text:
+                text += par.text.strip() + '\n'
+        if " " in text:
+            text = text.replace(" ", ' ')
+        self.article.text = text[:-1]
 
     def _fill_article_with_meta_information(self, article_soup):
-        pass
+        self.article.title = article_soup.find('h1').text
+        self.article.author = article_soup.find('div', class_="title-post").find(rel='author').text
+        self.article.date = self.unify_date_format(article_soup.find('div', class_="title-post").find('li').text)
 
     @staticmethod
     def unify_date_format(date_str):
         """
         Unifies date format
         """
-        pass
+        date = datetime.datetime(int(re.findall(r'\d{4}', date_str)[0]),
```

Comment:
too fragile, read more about interpreting dates in Python. find inspiration in the article module